site/content/troubleshooting-guide.md

```diff
@@ -1 +1,122 @@
-### TODO
\ No newline at end of file
+
+# Troubleshooting Guide
+
+This guide outlines some failure scenarios and their possible solutions.
+
+## Node.js not installed
+AWS.Deploy.Tools relies on AWS CDK to provision resources for the user's cloud application. AWS CDK requires Node.js to be installed in the user's machine.
```

Comment:
Throughout, I'd prefer you/your over user/user's.

CDK does in their guide - https://docs.aws.amazon.com/cdk/v2/guide/troubleshooting.html